python_env.yml

```diff
@@ -0,0 +1,25 @@
+name: python_nuclear_speckle_env
+channels:
+  - conda-forge
+dependencies:
+  - conda-forge::python=3.9.12
+  - conda-forge::jupyter=1.0.0
+  - conda-forge::pip
+  - conda-forge::opencv
+  - conda-forge::tifffile
+  - conda-forge::jupyterlab
+  - conda-forge::pandas=1.4.4
```

Comment:
Consider using later versions of pandas if possible (`>=2.x.x`).

Reply:
I can't remember why I set it as a specific pandas, but there might be a reason haha. I will need to test this in the future to see if anything breaks.